playwright/tests/customers.spec.ts

```diff
@@ -6,15 +6,20 @@ import { CUSTOMERS, CUSTOMER_ADDRESS } from "@data/e2eTestData";
 
 test.use({ storageState: "playwright/.auth/admin.json" });
 let customersPage: CustomersPage;
+let firstName: string;
+let lastName: string;
+let email: string;
+let note: string;
+
 test.beforeEach(({ page }) => {
   customersPage = new CustomersPage(page);
+  firstName = faker.name.firstName();
+  lastName = faker.name.lastName();
+  email = faker.internet.email();
+  note = faker.lorem.sentence();
```

Comment:
If you use faker in `beforeEach` and then try to use it in two different `test` then `firstname` will be the same in both cases. This is why I do not use it in `beforeEach`.